packages/core/src/components/forms/_file-upload.scss

```diff
@@ -69,20 +70,18 @@ $file-upload-button-width-large: $pt-grid-size * 8.5;
   top: 0;
```

Comment:
should probably just `@include overflow-ellipsis()` in this selector as @llorca suggests and be done with it. shouldn't _need_ to add this class to support long file names, the support should just be there.